### ISSUE TITLE
NET Core 3.0 support?

### ISSUE THREAD

**Issue by sander1095** [marektoman/mechaview#3]
Hello!

I have been playing around with your library this afternoon and it works really well! 

TL;DR: Will you make this library support .NET Core 3.0 because this will have WPF support?

Some background info on why I create this issue:

I had an issue with debugging because this is basically impossible I believe after the .exe is generated. The only way I could do this is to throw exceptions with some messages which would then be displayed as an error dialog and this is not too efficient.

The way I solved this is to create a .NET Framework console project which creates an instance of my viewmodel and then calls the "button" methods like your .exe would and this does allow me to just use my debugger and breakpoints, etc. Very useful:

```
public class MyViewModel
{
    [TextInput]
    public virtual string Name { get; set; }

    [Action]
    public void PerformButtonPress()
    {
        // ...
    }
}

public class Program 
{
        static void Main(string[] args)
        {
            var viewModel = new MyViewModel()
            {
                Name = "Foo"
            };

            try
            {
                viewModel.PerformButtonPress();
            }
            catch (Exception e)
            {
                // Inspect exceptions here if you don't break on user-managed exceptions
                throw;
            } 
        }
}
```

Sadly this "debugging" project can't be a .NET Core project because it calls my "mecha" project which is a .NET Framework 4.8 library project which, in my case, is doing some stuff like accessing the windows file system and unzipping files. 

I'd love to make my project .NET Core compliant and this can only be done if I could upgrade my mecha project to be a .NET Core 3.0 project.

**Comment by sander1095** [comment]
Actually, I just figured out you can just attach to the process in visual studio and debug that way.

Still, why not support .NET Core 3.0?
